package.json

```diff
@@ -30,29 +30,33 @@
         "@fontsource-variable/inter": "^5.0.18",
         "@ryangjchandler/alpine-tooltip": "^2.0",
         "@tailwindcss/cli": "^4.1.10",
-        "@tiptap/core": "3.0.0-beta.5",
-        "@tiptap/extension-blockquote": "3.0.0-beta.5",
-        "@tiptap/extension-bold": "3.0.0-beta.5",
-        "@tiptap/extension-code": "3.0.0-beta.5",
-        "@tiptap/extension-code-block": "3.0.0-beta.5",
-        "@tiptap/extension-document": "3.0.0-beta.5",
-        "@tiptap/extension-dropcursor": "3.0.0-beta.5",
-        "@tiptap/extension-heading": "3.0.0-beta.5",
-        "@tiptap/extension-image": "3.0.0-beta.5",
-        "@tiptap/extension-italic": "3.0.0-beta.5",
-        "@tiptap/extension-link": "3.0.0-beta.5",
-        "@tiptap/extension-list": "3.0.0-beta.5",
-        "@tiptap/extension-mention": "3.0.0-beta.5",
-        "@tiptap/extension-paragraph": "3.0.0-beta.5",
-        "@tiptap/extension-placeholder": "3.0.0-beta.5",
-        "@tiptap/extension-strike": "3.0.0-beta.5",
-        "@tiptap/extension-subscript": "3.0.0-beta.5",
-        "@tiptap/extension-superscript": "3.0.0-beta.5",
-        "@tiptap/extension-text": "3.0.0-beta.5",
-        "@tiptap/extension-underline": "3.0.0-beta.5",
-        "@tiptap/extensions": "3.0.0-beta.5",
-        "@tiptap/pm": "3.0.0-beta.5",
-        "@tiptap/suggestion": "3.0.0-beta.5",
+        "@tiptap/core": "3.0.0-beta.14",
+        "@tiptap/extension-blockquote": "3.0.0-beta.14",
+        "@tiptap/extension-bold": "3.0.0-beta.14",
+        "@tiptap/extension-code": "3.0.0-beta.14",
+        "@tiptap/extension-code-block": "3.0.0-beta.14",
+        "@tiptap/extension-document": "3.0.0-beta.14",
+        "@tiptap/extension-dropcursor": "3.0.0-beta.14",
```

Comment:
Does this still need to be installed if you pull it from the `extensions` package anyway?

Reply:
Could have sworn I removed it. But it is removed now as well as removing the '^' for the new extensions so that everything is locked to the same version.